docs/architecture/ADR-001-interest-stream.md

```diff
@@ -42,66 +38,43 @@ From [Cosmos IBC tutorial](https://tutorials.cosmos.network/understanding-ibc-de
 
-- Requirement: Umee chain stores uAsset <-> Asset exchange rate for each asset (not 1:1)
+- Requirement: Umee chain maintains `Token:Utoken` exchange rate for each token
 
-In this implementation, for each whitelisted Cosmos asset type, the Umee chain stores an "exchange rate" between the base asset and its associated uToken. The exchange rate starts equal to 1, and increases whenever interest would have been applied to uToken balances in the original implementation. Whenever a lender deposits or withdraws base assets for uToken, this exchange rate is used.
+In this implementation, for each token type accepted by the `x/leverage` module, the Umee chain computes an "exchange rate" between the base asset and its associated uToken. The exchange rate starts equal to 1, and increases as interest accrues. Whenever a lender deposits or withdraws base assets for uToken, the current exchange rate is used.
 
 Example scenario:
 
-> Two lenders Alice and Bob provide Atoms to the asset facility at different times and earn interest. Assume that for the duration of this scenario, the interest on deposited uAtoms is 0.1 percent per week (or 1 atom per week per 1000 deposited).
+> Two lenders Alice and Bob provide `ATOM` to the asset facility at different times and earn interest. Assume that for the duration of this scenario, the interest on deposited uAtoms is 0.1 percent per week (or `1 ATOM` per week per `1000 ATOM` deposited).
 >
-> The asset facility starts with 0 atoms in custody and 0 uAtoms in circulation. The exchange rate of Atom:uAtom starts at 1.
+> The asset facility starts with `0 ATOM` in custody and `0 u/ATOM` in circulation. The `u/ATOM` exchange rate starts at `1.0`.
 >
-> At a time we will label week=0, Alice deposits 2000 atoms and receives 2000 uAtoms per the exchange rate.
+> At a time we will label week=0, Alice deposits `2000 ATOM` and receives `2000 u/ATOM` per the exchange rate.
 >
-> At week=1, the exchange rate Atom:uAtom increases to 1.001. While Alice's uAtom balance is 2000, it is now worth 2002 Atoms if she were to redeem it.
+> By week=1, the `u/ATOM` exchange rate has increased to `1.001` due to interest. Alice's `2000 u/ATOM` are now worth `2002 ATOM` if redeemed.
 >
-> At week=2, Atom:uAtom increases to about 1.002 (technically 1.002001, but approximate amount will be displayed here)
+> By week=2, the `u/ATOM` exchange rate is `1.002` (technically `1.002001` due to compound interest, but approximate amount will be displayed here)
 >
-> Also at week=2, Bob deposits 1000 Atoms. Because the exchange rate has shifted, he received approximately 998 uAtoms, which are worth 1000 Atoms if he were to redeem them immediately.
+> Also at week=2, Bob deposits `1000 ATOM`. Because the exchange rate has shifted, he receives `998 u/ATOM`, which are worth `1000 ATOM` if he were to redeem them immediately.
```

Comment:
```suggestion
> Also at week=2, Bob deposits `1000 ATOM`. Because the exchange rate has shifted, he receives `998 u/ATOM`, which is worth `1000 ATOM` if he were to redeem them immediately.
```